junit-platform-console/src/main/java/org/junit/platform/console/options/Theme.java

```diff
@@ -79,43 +79,43 @@ public static Theme valueOf(Charset charset) {
 		this.blank = new String(new char[vertical().length()]).replace('\0', ' ');
 	}
 
-	public String root() {
+	public final String root() {
```

Comment:
Right, forgot about that possibility. In that case, I'm fine with declaring all methods `final`.